### DISCUSSION TITLE
Authentication problem still unsolved

### DISCUSSION THREAD

**Discussion by Zein-MB** [community/community#31816] [New to GitHub]
I've already seen many topics about this problem that pops up to me every time I try to push files onto my friend's repository, nevertheless no solutions could solve my issue, I have a personal access token and I use it when it asks me for the password but it still doesn't work
It throws the error:
remote: Permission to Zainmansour/ytdlProject.git denied to Zein-MB.
fatal: unable to access 'https://github.com/Zainmansour/ytdlProject.git/': The requested URL returned error: 403

I couldn't wisely choose what's the type of the help I need but I hope you to answer to my issue.
Have a good day!

**Comment by jfullstackdev** [comment]
do you have write access at your friend's repo ? if there is none, you must ask him/her first
